intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -216,6 +216,19 @@ \section{First Time connection to the \hpc}
 Are you sure you want to continue connecting (yes/no)? %\strong{\emph{yes}}%
 \end{prompt}
 
+A possible error message you can get if you previously saved your private key
+somewhere else than the default location (\verb|$HOME/.ssh/id_rsa|):
+
+\begin{prompt}
+%\shellcmd{Permission denied (publickey,gssapi-keyex,gssapi-with-mic).}%
+\end{prompt}
+
+In this case use:
+
+\begin{prompt}
+%\shellcmd{ssh -i path/to/location/of/private/key \userid{}@\loginnode{}}%
```

Comment:
@redfast00 Maybe it's better to use an absolute path here?

`/home/example/my_keys/id_rsa`